grouper/permissions.py

```diff
@@ -285,34 +262,37 @@ def filter_grantable_permissions(session, grants, all_permissions=None):
     return sorted(result, key=lambda x: x[0].name + x[1])
 
 
-def get_owners_by_grantable_permission(session, separate_global=False):
-    """
-    Returns all known permission arguments with owners. This consolidates
-    permission grants supported by grouper itself as well as any grants
-    governed by plugins.
+def get_owners_by_grantable_permission(
+    session: Session, separate_global: bool = False
+) -> Dict[object, Dict[str, List[Group]]]:
```

Comment:
As much as it hurts, I wonder if it wouldn't be less ugly to set `GLOBAL_OWNERS` to some string that isn't a valid permission name?

Reply:
It probably would, but I didn't want to make a behavior change like that, since it stands a high chance of introducing some obscure bug. I decided to leave it alone until we can fix it properly (which would involve returning permissions for global owners in a separate parallel structure and bundling them both together with a dataclass object).